assets/styles/layouts/_profile.scss

```diff
@@ -50,4 +50,8 @@ p.desc.col-md-8 {
 .btn:active {
   -webkit-box-shadow:none;
           box-shadow:none;
+}
+
+span.text-small {
+  font-size: 16px;
```

Comment:
would prefer to use the variable `font-size: $font-size` which is also currently at `16px`